index.html

```diff
@@ -5,21 +5,30 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
     <title>Portfolio</title>
   </head>
   <body>
```

Comment:
- [ ] Kindly ensure that while implementing other features of the website, your UI still looks intact as it is in the chosen Figma design. I noticed extra white spaces in your project while resizing the website (807px-1022px) which implies that your website is not very responsive. Kindly fix that and your website will look great 😄 

![image](https://user-images.githubusercontent.com/63473584/179811339-2ed9ce71-de70-4239-9502-42bd23668c66.png)